airflow/ui/src/components/ClearRun/ClearRunButton.tsx

```diff
@@ -57,11 +66,12 @@ const ClearRunButton = ({ dagId, dagRunId }: Props) => {
             requestBody: { dry_run: true, only_failed: onlyFailed },
           });
         }}
-        variant="outline"
+        size={withText ? "md" : "sm"}
+        variant={withText ? "outline" : "ghost"}
       >
-        <FiRefreshCw height={5} width={5} />
-        Clear Run
-      </Button>
+        <FiRefreshCw />
+        {withText ? "Clear Run" : ""}
```

Comment:
If the button displays as icon, no text is there. Can you please add a tooltip so that people can understand what the icon is for?

Reply:
Oh yeah, we should probably have an aria-label for accessibility

Reply:
Done thanks.
![Uploading Screenshot 2024-12-19 at 16.34.42.png…]()

Reply:
Added the aria label.